vignettes/wrapping-apis.Rmd

```diff
@@ -799,20 +837,22 @@ httr2 will automatically refresh the token in two cases:
 
 -   The token includes an expiry date that's in the past.
 
--   The request returns a 401 status code and
+-   The request returns a 401 status code.
+
+JENNY COMMENT HERE
```

Comment:
There seems to be some garbled / truncated wording in the bullet list above and the following 2 sentences.